src/scripts/main.js

```diff
@@ -1 +1,19 @@
 'use strict';
+
+const gallery = document.querySelector('.gallery');
+
+const changeImage = (eventElement) => {
+  const thumb = eventElement.target.closest('.gallery__thumb');
+
+  if(!thumb) {
```

Comment:
The condition in this `if` statement seems to be inverted. You're checking if `thumb` is falsy, but according to the variable name and the overall functionality of the method, you should execute the code inside this block when `thumb` is truthy. Consider changing this condition to `if(thumb)`.